.github/workflows/github_release.yaml

```diff
@@ -0,0 +1,133 @@
+name: Release to GitHub
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  wf_vars:
+    name: "Set Workflow Variables"
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ env.VERSION }}
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Set Base Environment Variables
+        run: |
+          echo >> $GITHUB_ENV VERSION="v-$(date +%Y.%m.%d-%H%M%S)"
+
+  build_router_gh_release:
+    name: "GitHub Release: Build Router"
+    needs:
+      - wf_vars
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: prime-router
+    env:
+      # These are for CI and not credentials of any system
+      POSTGRES_USER: prime
+      POSTGRES_PASSWORD: changeIT!
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+      - name: Set Environment Variables
+        run: |
+          echo >> $GITHUB_ENV DB_USER=${POSTGRES_USER}
+          echo >> $GITHUB_ENV DB_PASSWORD=${POSTGRES_PASSWORD}
+      - name: Build Router Package
+        run: bash ./build.sh -- gradle clean package
```

Comment:
This is just fetched from the current Azure release script here:
https://github.com/CDCgov/prime-reportstream/blob/master/.github/workflows/release.yml#L37

Should the Azure release be updated?

Reply:
It should; I'll create a separate issue/PR for that. But it's ok to change this to use `./gradlew clean package`.

Reply:
...PR created: https://github.com/CDCgov/prime-reportstream/pull/3311

Reply:
@cwinters-usds I confirmed this change caused today's failure. There must be additional changes needed when switching from `.build.sh` to `.gradlew`. My guess is solution is here https://github.com/CDCgov/prime-reportstream/blob/master/.github/workflows/build_hub.yml#L51-L64

#3311 will also need to updated, else it will fail as well

Reply:
#3339 Created with latest `gradlew` dependencies and `yarn` environment build